service/storage_service.cc

```diff
@@ -1566,7 +1567,8 @@ future<> storage_service::raft_bootstrap(raft::server& raft_server) {
                .set("topology_request", topology_request::join)
                .set("num_tokens", _db.local().get_config().num_tokens())
                .set("shard_count", smp::count)
-               .set("ignore_msb", _db.local().get_config().murmur3_partitioner_ignore_msb_bits());
+               .set("ignore_msb", _db.local().get_config().murmur3_partitioner_ignore_msb_bits())
+               .set("supported_features", _feature_service.supported_feature_set());
```

Comment:
We should somehow deal with this duplication so we don't have to remember to update so many places.
Out of the scope of this PR.